### PR TITLE
Documentation for go code

### PR DESCRIPTION
**Pull request by KeKsBoTer**
Removed unused function "GetMessagesForUser"
Added timestamp to messages

### CODE REVIEW

File: cmd/eve/config.go [hunk 1]
```diff
@@ -20,6 +20,7 @@ type Config struct {
 // 		host: "" (empty)
 //		http: 80
 //		https: 443
+// programm usage e.g.: eve -host eve.de -http 80 -https 443
```
Comment (niklaskorz) [inline cmd/eve/config.go]: program, not programm

File: db/db.go [hunk 2]
```diff
@@ -126,18 +137,34 @@ const (
 	Female = Gender(1)
 )
 
-// Bot represents database entry of a bot
+// Bot represents a bots entry in the database
+// A bot is only accesable for one user
```
Comment (niklaskorz) [inline db/db.go]: accessible

File: db/db.go [hunk 3]
```diff
@@ -165,19 +192,33 @@ const (
...
+
+	// The text that was sent
+	Content string
+
+	//
+	Rating float64
 }
 
-// MessageMaxLength defines the maximum message length
+// MessageMaxLength defines the maximum message length that a user can sent to a bot
```
Comment (niklaskorz) [inline db/db.go]: send, not sent

File: manager/bots/pool.go [hunk 4]
```diff
@@ -5,12 +5,17 @@ import (
...
+// A task holds message as input and a channel to send the answer to
+// Every request to a bot needs all the information stored in message data.
 type botTask struct {
 	data   MessageData
 	result chan BotAnswer
 }
 
-//BotPool manages a pool of bots
+// BotPool manages a pool of running instances of the bot python script.
+// In order to generate the bot answers faster for multiple users, any incomming request is forwarded to a free bot instance.
```
Comment (niklaskorz) [inline manager/bots/pool.go]: incoming

File: manager/bots/pool.go [hunk 5]
```diff
@@ -5,12 +5,17 @@ import (
...
+// For every incomming request a task is generated and executed by the next free instance.
```
Comment (niklaskorz) [inline manager/bots/pool.go]: incoming

File: manager/bots/pool.go [hunk 6]
```diff
@@ -20,6 +25,7 @@ type BotPool struct {
 }
 
 // NewBotPool creates a new pool with a given amount of bot instances
+// The task buffer size is 128. This means the pool can have up to 128 tasks in the que.
```
Comment (niklaskorz) [inline manager/bots/pool.go]: queue

File: manager/bots/pool.go [hunk 7]
```diff
@@ -29,7 +35,8 @@ func NewBotPool(size int) *BotPool {
 	return pool
 }
 
-// HandleRequest handels a message request and redirects it to a free bot instance
+// HandleRequest takes incoming requests and makes the task work off by the next free bot instance.
```
Comment (niklaskorz) [inline manager/bots/pool.go]: "makes the task work off"?

File: manager/bots/pool.go [hunk 8]
```diff
@@ -29,7 +35,8 @@ func NewBotPool(size int) *BotPool {
...
+// The function always returns a answer with text. If an error occures errorBotAnswer(...) is returned.
```
Comment (niklaskorz) [inline manager/bots/pool.go]: an answer
occurs

File: manager/bots/pool.go [hunk 9]
```diff
@@ -84,7 +90,9 @@ func (p *BotPool) worker() {
 	}
 }
 
-//Resize changes the count of bot instances
+// Resize changes the count of bot instances
+// The size can be any positiv number including zero.
```
Comment (niklaskorz) [inline manager/bots/pool.go]: positive

File: manager/bots/pool.go [hunk 10]
```diff
@@ -84,7 +90,9 @@ func (p *BotPool) worker() {
...
+// Decreasing the size can take a while, because only bot instances that are not currently working, can be destroyed.
```
Comment (niklaskorz) [inline manager/bots/pool.go]: Last comma is unnecessary

File: manager/message.go [hunk 11]
```diff
@@ -8,9 +8,12 @@ import (
 	"github.com/AAA-Intelligence/eve/manager/bots"
 )
 
+// Global bot pool that is used for all requests
 var botPool *bots.BotPool
 
-// takes incomming message requests, sends them to the bot instance and returns the answer
+// Takes incomming message requests, sends them to the bot instance and returns the bots answer
```
Comment (niklaskorz) [inline manager/message.go]: incoming

Comment (niklaskorz) [reply manager/message.go]: the bot's answer

File: manager/message.go [hunk 12]
```diff
@@ -8,9 +8,12 @@ import (
...
+// All messages are stored in the database int the Message table.
```
Comment (niklaskorz) [inline manager/message.go]: in the Message table, not int

File: manager/message.go [hunk 13]
```diff
@@ -8,9 +8,12 @@ import (
...
+// If any error occures the string "Ok" is returned
```
Comment (niklaskorz) [inline manager/message.go]: occurs

File: manager/webserver.go [hunk 14]
```diff
@@ -135,21 +150,26 @@ func StartWebServer(host string, httpPort int) {
 }
 
 // GetBotFromRequest checks if there is a bot id in the request (HTTP GET e.g. ?bot=2)
-// If not the first bot for the user is returned
+// Also it checks if the bot belongs to the authenticated user
```
Comment (niklaskorz) [inline manager/webserver.go]: It also checks

Comment (KeKsBoTer) [reply manager/webserver.go]: Thx

**Pull request merged**